roles/openshift_facts/tasks/main.yml

```diff
@@ -6,5 +6,8 @@
     - ansible_version | version_compare('1.9.0', 'ne')
     - ansible_version | version_compare('1.9.0.1', 'ne')
 
+- name: Ensure python-netaddr is installed
+  yum: pkg=python-netaddr state=installed
```

Comment:
@sdodson It'd be nice to keep the subnet calculation if possible...  I haven't tested, but you could probably do something like this in openshift_facts:
```
if atomic:
    first_svc_ip = do_some_subnet_guesstimation(facts['master']['portal_net'])
else:
    from netaddr import IPNetwork
    first_svc_ip = str(IPNetwork(facts['master']['portal_net'])[1])
```